docs/components/stepper.md

```diff
@@ -0,0 +1,130 @@
+# Stepper
+
+**KStepper** - A basic Stepper component
+
+<KStepper :steps="defaultItems" />
+
+```html
+<KStepper :steps="steps" />
+```
+
+## Props
+
+### steps
+
+An array of `KSteps`. Each `KStep` has a `label` and an optional `state`.
+
+<div>
+  <KStepper :steps="[
+      { label: 'Step  a long long long long time ago', state: 'completed' },
+      { label: 'in a galaxy far far away', state: 'completed' },
+      { label: 'Kongponents were battling in space and', state: 'pending' },
+      { label: 'fighting robots and space monsters with lots of explosions' }
+    ]"
+  />
+</div>
+
+```html
+<KStepper :steps="[
+    { label: 'Step  a long long long long time ago', state: 'completed' },
+    { label: 'in a galaxy far far away', state: 'completed' },
+    { label: 'Kongponents were battling in space and', state: 'pending' },
+    { label: 'fighting robots and space monsters with lots of explosions' }
+  ]"
+/>
+```
+
+### maxLabelWidth
+
+The width of step labels (default is `170`). Currently we support numbers (will be converted to `px`), `auto`, and percentages for values.
+
+<KStepper :steps="longSteps" max-label-width="100" />
+
+```html
+<KStepper :steps="steps" max-label-width="100" />
+```
+
+## KStep
+
+**KStepper** generates a **KStep** for each item in the `steps` property. Each step has a `label`, `state`, and an associated divider (unless it's the last step.)
+
+### Properties
+
+- `label` (required) - the text displayed beneath the step
+- `state` - the state of the step controls the icon, we support: `completed`, `pending`, and `error`
```

Comment:
```suggestion
- `state` - the state of the step controls the icon, we support: `completed`, `pending`, and `error`. If a 'state' is not provided, it will show the default grey icon.
```